project_profile_counter.py

```diff
@@ -0,0 +1,28 @@
+import json
+import requests
+import collections
+
+SERVER = 'https://tokyo.sdelements.com'
+API_TOKEN = 'f258c47557a3f98f55d4fbd0cb9d8354c86fbb52'
+PROJECTS_URL = '{server}/api/v2/projects/?page_size=100'.format(server=SERVER)
+REQUESTS_HEADER = {
+    # Authorization header + generated API TOKEN
+    'Authorization': "Token " + API_TOKEN,
+    'Accept': "application/json"             # Accepts API response in JSON format
+}
+url = PROJECTS_URL
```

Comment:
Blank line above here separates these variables used by the loop below from the constants above.